Dockerfile

```
@@ -1,8 +1,9 @@
 #base Build
 ARG node_version=10
 FROM node:$node_version as base_couch2pg_build
-RUN apt-get update
-RUN apt-get -y install postgresql-client curl
+RUN apt update
+RUN apt dist-upgrade -y
```

```suggestion
RUN apt full-upgrade -y
```

Not sure about Dockerfile best practices, but in general I would lean towards `full-upgrade` instead of `dist-upgrade`.

---

Interesting! TIL - they're the same thing (see [supersuer](https://superuser.com/questions/1554163/what-is-the-difference-between-apt-dist-upgrade-and-apt-upgrade) and [askubuntu](https://askubuntu.com/questions/770135/apt-full-upgrade-versus-apt-get-dist-upgrade) for rando citations).